src/Combobox/index.js

```diff
@@ -121,6 +135,7 @@ const Combobox = ({
   errorText,
   icon,
   testId,
+  renderEndContent = defaultRenderEndContent,
```

Comment:
**Optional**

Renaming this `endContent` would make it easier to remember (although types will help in the future anyway)

Reply:
oh, that's why:

```
* Signature: `(isOpen) => React.ReactNode`
```